### DISCUSSION TITLE
proxy

### DISCUSSION THREAD

**Discussion by obkduishinijf** [3kh0/ext-remover#583] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
https://link-target.net/644665/unblock648 (i know it's a linkvertise, use it and help me get enough money to boost the glitch website)

**Comment by bartholemuew** [comment]
i aint using no linkvertise 💀

**Reply by GGboogie** [reply]
This works bruh even though it seems kinda sus. Also, why haven't they blocked "linkvertise" yet? I'm very surprised

**Reply by GermanicPanzer** [reply]
use bypass.vip to bypass linkvertise
it works but make sure to close out the pop up tabs as quick as you can, it takes around 3 pop ups for it to work

**Reply by amelia-bartlett** [reply]
> This works bruh even though it seems kinda sus. Also, why haven't they blocked "linkvertise" yet? I'm very surprised

gonna have to find the patch for universal bypass that re-adds linkvertise support

**Reply by amelia-bartlett** [reply]
Ok, here is what it links to: https://first-neon-honeycup.glitch.me/

**Reply by GGboogie** [reply]
Hey, what happened?:
![Screenshot 2023-04-20 10 36 24 AM](https://user-images.githubusercontent.com/97118713/233444585-6a06eb99-d9e4-467d-9378-313e2102795b.png)
And also when I click on the linkvertise button it shows me this:
![Screenshot 2023-04-20 10 35 35 AM](https://user-images.githubusercontent.com/97118713/233444692-8ae75d33-0f1e-41bf-8dfc-61a530fcd32e.png)

**Reply by py660** [reply]
same with mine: glitch is cracking down on proxies as well.

**Reply by obkduishinijf** [reply]
> Hey, what happened?: ![Screenshot 2023-04-20 10 36 24 AM](https://user-images.githubusercontent.com/97118713/233444585-6a06eb99-d9e4-467d-9378-313e2102795b.png) And also when I click on the linkvertise button it shows me this: ![Screenshot 2023-04-20 10 35 35 AM](https://user-images.githubusercontent.com/97118713/233444692-8ae75d33-0f1e-41bf-8dfc-61a530fcd32e.png)

glitch started cracking down on proxies, can't use glitch for 'em anymore so good luck
